### DISCUSSION TITLE
Multiple supabase projects, one user pool like AWS Cognito etc.

### DISCUSSION THREAD

**Discussion by Zyles** [supabase/supabase#16673] [Questions]
Is it possible to run multiple supabase projects / databases utilizing one user provider such as AWS Cognito or Google Identity Platform for example?

What I want to idealy do is centralize user management over multiple supabase databases (one per tenant).

Identify platform would store which tenant(s) the user belongs to and my app would then auth against that supabase instance which the user is requesting.

This will also help me with having "super admins" that can access multiple projects.

But how would that work with auth and ownership within supabase?

**Comment by GaryAustin1** [comment]
Supabase database thru the API just cares about the jwt being valid (matching its secret and not expired).  You can make calls just setting the Authorization header in fetch or Supabase client with those jwts.  If you used the same jwt secret for all projects and your jwt auth generator then a user jwt would work on any instance they have data in and meet the RLS.  RLS could still be on the sub with auth.uid() if it is a UUID or any other claims in the jwt.

I've not done this and don't have an example.
